css-two-column-layout/styles.css

```diff
@@ -0,0 +1,75 @@
+body {
+  font-family: 'Karla', sans-serif;
+}
+
+.container {
+  max-width: 1000px;
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+  flex-direction: column;
+}
+
+h1 {
+  font-weight: 700;
+  font-size: 3.2rem;
+  margin: 1.4rem 0;
+  letter-spacing: 0.2rem;
+}
+
+h1,
+h3,
+button {
+  text-transform: uppercase;
+}
+
+h3 {
+  font-size: 1.4rem;
+  margin-top: 2.8rem;
+  margin-bottom: 1rem;
+}
+
+p {
+  line-height: 1.7rem;
+  margin: 0;
+}
+
+.heading-container,
+.button-container {
+  text-align: center;
+}
+
+.heading-container {
+  margin: 2rem 10rem;
```

Comment:
Rather than setting such a large margin, you should limit the width of this element, and set `margin: auto` to center the content in the leftover space.